Assignments/assignment02_lottery/lottery.py

```diff
@@ -8,26 +8,29 @@
 MSG_INPUT_LOT_NUM = "> 지난 주 당첨 번호를 입력해주세요: "
 MSG_RESULT = "> 로또 당첨 결과"
 
+
 def main():
     run()
 
+
 # 얼마 구입할 건지 입력
 def input_purchase_price():
     purchase_price = input(MSG_PURCHASE)
-    while(purchase_price.isdigit() == False) :
+    while (purchase_price.isdigit() == False):
```

Comment:
아래와 같이 심플하게 바꿀 수 있을 것 같아요!!!

```suggestion
    while not purchase_price.isdigit():
```

Reply:
세훈님! 꼼꼼하게 리뷰해주셔서 정말 감사합니다. 
누군가에게 이렇게 디테일하게 피드백 받은 적이 처음이에요. 
기초적인 학부수업 두어개를 제외하고는 프로그래밍을 체계없이 후루룩 배워서 코드 최적화에 대한 갈증이 컸었습니다. 
아무래도 혼자서 학습하다보니 작성한 코드에 기능적으로는 문제가 없더라도 분명히 존재할 개선점들에 대해서는 놓치는 부분들이 많다고 느꼈거든요.
협업 시 암묵적인 룰들에 대해, 그리고 차근차근 배워온 전공자들은 어떤 식으로 코드를 작성하는지에 대해 궁금한 점이 많았는데 활동 기간 동안 많은 걸 배워갈 수 있을 것 같습니다.
멋사 지원하길 잘했다는 생각이 다시 한 번 드네요..! ㅎㅎ

깃헙 이용은 제가 아직 익숙치 않아서ㅠㅠ 머지, 풀리퀘스트 기능은 시험기간 끝나고 좀 더 공부한 후 적용해보겠습니다. 
지적해주신 부분들 반영해서 다시 커밋, 푸시하겠습니다. 
다시 한번 감사합니다~

//
while not은 한번도 이용해본 적이 없어서 생각을 못했었네요.
앞으로 while not도 적재적소에 이용해보겠습니다!

Reply:
0원 예외처리는 깜빡하고 빠뜨렸네요,ㅜㅜ 
다시 커밋 / 푸시 할 때 추가해보겠습니다.!